### PR TITLE
http reference is changed in documentation

### PR DESCRIPTION
**Pull request by ManognaMutyala**
closes #

addresses #

## What is included in this PR?
Https reference for ODK Briefcase was provided earlier and now reference has been changed to the ODK Briefcase doc.

## What new issues will need to be opened because of this PR?
No new issues

### CODE REVIEW

File: aggregate-deployment-planning.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ ODK Aggregate can be deployed to Google's App Engine, Amazon's EC2 cloud service
 
 In general, you should gain familiarity with ODK Aggregate using Google's App Engine cloud services or the ODK Aggregate VM before attempting an EC2 or Tomcat deployment. See :ref:`Aggregate installation <install-app-engine>`.
 
-Once you can download forms from ODK Aggregate to :ref:`ODK Collect <collect-introduction>`, fill them out, upload finalized forms to ODK Aggregate, view, export and publish data from within ODK Aggregate, and download data from ODK Aggregate using `ODK Briefcase <http://docs.opendatakit.org/briefcase-guide/>`_, you might then consider whether a Google App Engine cloud services deployment or VM is appropriate for your needs.
+Once you can download forms from ODK Aggregate to :ref:`ODK Collect <collect-introduction>`, fill them out, upload finalized forms to ODK Aggregate, view, export and publish data from within ODK Aggregate, and download data from ODK Aggregate using See  :doc: briefcase-guide, you might then consider whether a Google App Engine cloud services deployment or VM is appropriate for your needs.
```
Comment (adammichaelwood) [inline aggregate-deployment-planning.rst]:
```
:doc:`briefcase-guide`
```

Comment (adammichaelwood) [reply aggregate-deployment-planning.rst]: Also, the sentence no longer makes sense with the word "See" in it.

**Comment by adammichaelwood** [comment]
Thanks!

File: aggregate-deployment-planning.rst (hunk 1 above)
Comment (ManognaMutyala) [reply aggregate-deployment-planning.rst]: Thanks for the info sir and I will take care of those details from now.

**Pull request merged**